core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -171,8 +171,8 @@ public void add(AlluxioURI alluxioUri, AlluxioURI ufsUri, long mountId, MountOpt
         if ((ufsUri.getScheme() == null || ufsUri.getScheme().equals(mountedUfsUri.getScheme()))
             && (ufsUri.getAuthority() == null || ufsUri.getAuthority()
             .equals(mountedUfsUri.getAuthority()))) {
-          String ufsPath = ufsUri.getPath();
-          String mountedUfsPath = mountedUfsUri.getPath();
+          String ufsPath = ufsUri.getPath().isEmpty() ? "/" : ufsUri.getPath();
```

Comment:
ditto, put the TODO here linking to the JIRA, so we know why the code is like this